release-schema.json

```diff
@@ -86,6 +86,56 @@
             "string",
             "null"
           ]
+        },
+        "hasExclusiveRights": {
+          "title": "Has exclusive rights",
+          "description": "The procuring entity grants the contractor the right to perform the public service described in the contract free of competition.",
```

Comment:
I wanted to make it more generic, but we can stick to the regulation.

We can make also make it shorter:

> A right entitling a public service operator to operate certain public services to the exclusion of any other operator.

How generic/specific the definitions should be?

Reply:
Hmm, the semantics change if we omit "on a particular route", etc. as it implies that the exclusion is universal, rather than scoped. Let's leave it as-is. It's always easier to relax semantics in the future, than to tighten them up again.